### PR TITLE
contrib: update journal field labels

### PR DESCRIPTION
**Pull request by tmorrell**
:heart: Thank you for your contribution!

### Description

This PR changes the label for the pages field in the deposit form. The motivation is that users misinterpret the field, by putting in incorrect metadata (total number of physical pages) or not understanding that article numbers can also be put into this field. An example of some issues:

Article number in the issue field and total number of pages in pages field [Search for radio jets from massive young stellar objects. Association of radio jets with H2O and CH3OH masers](https://zenodo.org/record/4438759) 

Missing article number [Visible and near-infrared spectro-interferometric analysis of the edge-on Be star o Aquarii](https://zenodo.org/record/4673798) 

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [ ] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [ ] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [ ] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [ ] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [ ] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [ ] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [ ] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [ ] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

**Comment by tmorrell** [comment]
Screenshot 2023-04-21 at 1 57 02 PM

**Review by fenekku — APPROVED** [review]
LGTM

**Comment by kpsherva** [comment]
was this discussed with the community, since it is influencing all the instances?

File: invenio_rdm_records/contrib/journal/custom_fields.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def mapping(self):
                     "description": "",
                 },
                 "pages": {
-                    "label": _("Pages"),
+                    "label": _("Page Range or Article Number"),
```
Comment (kpsherva) [inline invenio_rdm_records/contrib/journal/custom_fields.py:98]: to keep it consistent with other fields' labels:

```suggestion
                    "label": _("Page range or article number"),
```

**Comment by tmorrell** [comment]
Screenshot 2023-04-25 at 10 48 28 AM

I'm not sure what type of community discussion would be best...since it's a small change but it does impact everyone using these fields. Anyone who is collecting metadata on journals is going to run across article numbers. This proposal makes the label more accurate for how the `pages` field is implemented.

I'm happy to have this PR be an example for the PR community group to discuss. What level of review is required for these types of interface changes?

**Comment by lnielsen** [comment]
The change makes sense to me and indeed many new articles have an article number instead of page range, and agree that the previous label could be misunderstood. Also, perhaps it would be good to add placeholder text to show example of the data input.

**Comment by tmorrell** [comment]
Thanks! I've gone and added some more placeholder text to the journals fields, so this needs another round of review.
Screenshot 2023-04-26 at 4 06 21 PM

**Pull request merged**